### PR TITLE
fix: product card image blocks

### LINKED ISSUE #2573: Next Image tag vs HTML img tag
### Provide a description of requested docs changes

Hey, in the docs we have a misaligned usage of Next `Image` tag and HTML `img` tag in Blocks section:

E.g.

NavBarTop - https://docs.storefrontui.io/v2/react/blocks/NavbarTop.html#navbartop-with-white-background

```
<a href="/" aria-label="SF Homepage" className="inline-block text-primary-700">
    <img src={brandLogo.src} alt="Sf Logo" className="hidden lg:block mr-10 w-[12.5rem] h-[1.75rem]" />
     <img src={brandLogoSign.src} alt="Sf Logo" className="block w-8 h-8 mr-4 md:w-10 md:h-10 lg:hidden" />
</a>
```

ProductCard - https://docs.storefrontui.io/v2/react/blocks/ProductCard.html#productcard-vertical

```
<SfLink href="#">
          <Image
            src={productImage.src}
            alt="Great product"
            className="object-cover h-auto rounded-md aspect-square"
            width={300}
            height={300}
          />
        </SfLink>
```

I believe we should decide on what to use. IMO, we should stick with HTML `img` tag, since not all users are using Next.js

### Able to fix / change the documentation?

- [ ] Yes
- [ ] No

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by runyasak**
# Related issue

Closes #2573 

# Scope of work

I implemented `img` instead of `Image` from `next/image` on `ProductCardHorizontal` and `ProductCardVertical`.

# Screenshots of visual changes

# Checklist

- [X] Self code-reviewed
- [X] Changes documented
- [X] Semantic HTML
- [X] SSR-friendly
- [X] Caching friendly
- [X] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] blocks created
- [ ] cypress tests created

### CODE REVIEW

**Review by Szymon-dziewonski — APPROVED** [review]
Thank you for this change, could you only please change PR message that will follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), so in your case it would be `fix: ProductCard block Image tag change` , Thank you :)

**Comment by runyasak** [comment]
> Thank you for this change, could you only please change PR message that will follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/), so in your case it would be `fix: ProductCard block Image tag change` , Thank you :)

Hi, @Szymon-dziewonski.
Sorry for my commit message. I just put an explanation of what I did in the commit message, without following Conventional Commits.
That's would be a great suggestion for me, I will learn more and do it better.

By the way, I've updated the commit message as your suggestion already.

**Comment by Szymon-dziewonski** [comment]
@runyasak no worries, my mistake I was not precise enough I meant PR title, which will be commit message after merge with squash. Your commit messages will not affect end result since we do `squash and merge`, which means all your commits will be combined into one commit with message as your Pull request title. 

In short PR title has wrong format at this moment and need to be written in conventional commit standard :)

image

**Review by aniamusial — APPROVED** [review]
nice!

**Comment by runyasak** [comment]
> @runyasak no worries, my mistake I was not precise enough I meant PR title, which will be commit message after merge with squash. Your commit messages will not affect end result since we do `squash and merge`, which means all your commits will be combined into one commit with message as your Pull request title.
> 
> In short PR title has wrong format at this moment and need to be written in conventional commit standard :)
> 
> image

@Szymon-dziewonski Thank you for your comments and for updating my PR title. 😁

> nice! Also, remember to use smallcase for PR titles :) We switched from "showcase" naming to "blocks" too.

@aniamusial Thank you for your comments. I've updated to `blocks` as your suggestion. 😁

**Comment by Szymon-dziewonski** [comment]
@runyasak your welcome, at this point once we will fix our pipeline I will merge your Pull Request, once again thank you for your contribution :)

**Pull request merged**
